### PR TITLE
Inform user on which roles are removed

### PR DESCRIPTION
**Pull request by tavy88**
We want to inform the users which role they remove
by displaying a flash message.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions
https://www.w3.org/TR/WCAG20-TECHS/ARIA19.html

Design info
https://github.com/alphagov/govuk-design-system-backlog/issues/2

Screen Shot 2019-10-31 at 13 59 39

https://dfedigital.atlassian.net/browse/GET-658

### CODE REVIEW

File: app/views/skills/index.html.erb [hunk 1]
```diff
@@ -13,6 +13,9 @@
   <div class="govuk-grid-column-two-thirds">
     <div class="govuk-grid-column-full govuk-!-padding-0">
       <h1 class="govuk-heading-xl">Your skills</h1>
+      <% if flash[:notice].present? %>
+        <div role="alert" class="alert-info" aria-live="polite"><%= flash[:notice] %></div>
```
Comment (tavy88) [inline app/views/skills/index.html.erb]: Please check the attached resources.

File: app/views/skills/index.html.erb [hunk 2]
```diff
@@ -13,6 +13,9 @@
   <div class="govuk-grid-column-two-thirds">
     <div class="govuk-grid-column-full govuk-!-padding-0">
       <h1 class="govuk-heading-xl">Your skills</h1>
+      <% if flash[:notice].present? %>
+        <div role="region" class="alert-info" aria-live="polite"><%= flash[:notice] %></div>
+      <% end %>
```
Comment (tavy88) [inline app/views/skills/index.html.erb]: Please check the attached links in the PR's description.

File: app/controllers/job_profiles_controller.rb [hunk 3]
```diff
@@ -14,7 +14,7 @@ def destroy
     user_session.remove_job_profile(resource.id)
 
     redirect_to(
-      skills_path(query_params)
+      skills_path(query_params), flash: { notice: "The #{job_profile.name.downcase} role has been removed." }
```
Comment (smoothcontract) [inline app/controllers/job_profiles_controller.rb]: I like to use locale entries for flash messages if possible - it keeps the controller looking cleaner. Could you add an entry for this please?

Comment (tavy88) [reply app/controllers/job_profiles_controller.rb]: @smoothcontract updated!

File: app/views/skills/index.html.erb (hunk 2 above)
Comment (smoothcontract) [reply app/views/skills/index.html.erb]: If we end up using this flash pattern on other pages then this markup could be moved to a partial, but I'm happy for it to be inline on this page for now 👍

File: config/locales/en.yml [hunk 4]
```diff
@@ -190,3 +190,7 @@ en-GB:
               blank: Enter a postcode
             dob:
               invalid: Enter a valid date of birth
+
+  job_profiles:
+    destroy:
+      notice: The %{name} role has been removed.
```
Comment (smoothcontract) [inline config/locales/en.yml]: 👍

**Pull request merged**